README.md

```diff
@@ -144,6 +144,7 @@ The script also supports branches automatically. If you work in a checked out br
 There are two workflows for develompent provided:
 1) Development mode - work in the feature branch, apply changes related to your fork, revert the changes when the work is done
 2) Preview mode - work in a feature branch, apply script which creates new preview branch and create additional commit with for customization
+3) e2e mode - Same as `preview` mode, but also installs RHSSO with preconfigured realm to be used with toolchain operators.
```

Comment:
not sure about name of the mode, e2e is overused, in this case there is just sso 'addon', so maybe `sso mode`?

Reply:
My wishful thinking for the future with this mode is to be able to deploy HAC as well. Then this mode would contain whole appstudio experience (HAC+SSO+Appstudio) on single (ephemeral) cluster. Not sure how possible is that though.